web/themes/custom/pineapple/templates/content/node--page--full.html.twig

```diff
@@ -0,0 +1,126 @@
+ *   content tag that appears in the template.
+ * - author_attributes: Same as attributes, except applied to the author of
+ *   the node tag that appears in the template.
+ * - title_prefix: Additional output populated by modules, intended to be
+ *   displayed in front of the main title tag that appears in the template.
+ * - title_suffix: Additional output populated by modules, intended to be
+ *   displayed after the main title tag that appears in the template.
+ * - view_mode: View mode; for example, "teaser" or "full".
+ * - teaser: Flag for the teaser state. Will be true if view_mode is 'teaser'.
+ * - page: Flag for the full page state. Will be true if view_mode is 'full'.
+ * - readmore: Flag for more state. Will be true if the teaser content of the
+ *   node cannot hold the main body content.
+ * - logged_in: Flag for authenticated user status. Will be true when the
+ *   current user is a logged-in member.
+ * - is_admin: Flag for admin user status. Will be true when the current user
+ *   is an administrator.
+ *
+ * @see template_preprocess_node()
+ *
+ * @todo Remove the id attribute (or make it a class), because if that gets
+ *   rendered twice on a page this is invalid CSS for example: two lists
+ *   in different view modes.
+ */
+#}
+{%
+  set classes = [
+    'node',
+    'node--type-' ~ node.bundle|clean_class,
+    node.isPromoted() ? 'node--promoted',
+    node.isSticky() ? 'node--sticky',
+    not node.isPublished() ? 'node--unpublished',
+    view_mode ? 'node--view-mode-' ~ view_mode|clean_class,
+  ]
+%}
+{{ attach_library('classy/node') }}
+<article{{ attributes.addClass(classes) }}>
+  {% if not content.field_bp_hero_image|field_value %}
+    {{ title_prefix }}
+    {% if label %}
+     <h1{{ title_attributes.addClass('page-title') }}>
+      {{ label }}
+      </h1>
+    {% endif %}
+    {{ title_suffix }}
+  {% endif %}
+  {% if display_submitted %}
+    <footer class="node__meta">
+      {{ author_picture }}
+      <div{{ author_attributes.addClass('node__submitted') }}>
+        {% trans %}Submitted by {{ author_name }} on {{ date }}{% endtrans %}
+        {{ metadata }}
+      </div>
+    </footer>
+  {% endif %}
+
+  <div{{ content_attributes.addClass('node__content') }}>
+    {% if content.field_bp_hero_image|field_value %}
+      {% set hero_url = bamboo_render_image_style(content.field_bp_hero_image|field_target_entity.field_media_image.entity.uri.value, 'hero_image_1265_x_527') %}
+      <div class="hero-container breakout">
+        <div class="hero-image-container" style="background-image: url({{ hero_url }})">
+        </div>
+        <div class="hero-image-text container">
+          <div class="article-title">
+            <h1>
+              {{ label }}
+            </h1>
+          </div>
+          <div class="author-name">
+            <p>by: {{ node.Owner.name.value }}</p>
+          </div>
+        </div>
+      </div>
+    {% endif %}
+    <div class="top-spacer">
```

Comment:
Add closing div tag for this div

Reply:
Rename class name to something else. 
http://bdavidxyz.com/blog/how-to-name-css-classes/

Reply:
Done.